### ISSUE TITLE
Docker ARM support

### ISSUE THREAD

**Issue by jmreicha** [vmware-tanzu/velero#720]
**Describe the solution you'd like**

A quick looks at the manifest for the ark Docker images show that amd64 is currently the only supported architecture.

```
./manifest-tool inspect gcr.io/heptio-images/ark
gcr.io/heptio-images/ark: manifest type: application/vnd.docker.distribution.manifest.v2+json
      Digest: sha256:d883bb51599bdae48b022a1e5e52d153c61c4ee6d05108450a283135ce47f1ae
Architecture: amd64
          OS: linux
    # Layers: 4
      layer 1: digest = sha256:911c6d0c7995e5d9763c1864d54fb6deccda04a55d7955123a8e22dd9d44c497
      layer 2: digest = sha256:769ba40b5bd2a959af291441adff89b5702ddc039027262ffe9363a222d6c4c1
      layer 3: digest = sha256:82f011c5f4db49c069cc3066e3ba6cd37b67937269fa1734a901a49a2f2a8f55
      layer 4: digest = sha256:1160930f859d360b7015bc8a1894623e7f49f3a77be433f0d179a4e3c432a5ea
```

It would be nice to have multiarch builds for Ark, including arm and arm64.

**Comment by skriss** [comment]
Thanks for reporting this @jmreicha, we'll take a look!

**Comment by vielmetti** [comment]
Any progress on this? I am aware of someone who tried to use this on arm64 and ran into failures, and I know that Docker has improved multiarch build support since this issue was originally brought up.

**Comment by mylesagray** [comment]
I had a go at it and built multiarch images based on Ubuntu Xenial and Alpine for `amd64`, `arm32v6`, `arm32v7`, `arm64v8`.

Here: http://github.com/mylesagray/velero-multiarch

**Comment by prydonius** [comment]
see also #1645, @skriss should one of these be closed as a dupe?

**Comment by mylesagray** [comment]
I think this provides a basis for the `ppc64le` work in there too - could track in either, but my PoC is just adding arm, no ppc as there are no published bins for it.

**Comment by skriss** [comment]
Once we get #1645 in, it should be a quick change to add ARM builds.

**Comment by onedr0p** [comment]
Would this provide multiarch Docker images, or separate Docker image per arch?

**Comment by skriss** [comment]
we've now got multiarch support via Docker manifest lists merged, so if anyone wants to work on adding arm builds, it should be ready to go!

**Comment by shaneutt** [comment]
I got a start on it and put up a draft PR @ https://github.com/vmware-tanzu/velero/pull/2227, but it's late here and I need to get to bed so I'll come back around to this soon.

**Comment by skriss** [comment]
closed by #2227
